mitosheet/mitosheet/tests/public/v3/sheet_functions/string_functions/test_concat.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Saga Inc.
+# Distributed under the terms of the GPL License.
+
+from typing import Any
+import pytest
+import pandas as pd
+import numpy as np
+from mitosheet.public.v3.rolling_range import RollingRange
+from datetime import datetime
+
+from mitosheet.public.v3.sheet_functions.string_functions import CONCAT
+
+
+CONCAT_VALID_TESTS: Any = [
+    # Just constant tests
+    (['a', 'b'], 'ab'),
+    (['a', 1, True], 'a1True'),
+
+    # Constants and series
+    (['a', pd.Series(['a', 'b', 'c'])], pd.Series(['aa', 'ab', 'ac'])),
```

Comment:
Can we add a `None` test here? It probably makes sense to make sure we have at least one None test per function.